packages/aila/src/lib/agents/compatibility/streamHandling.ts

```diff
@@ -0,0 +1,268 @@
+// packages/aila/src/lib/agents/streamHandling.ts
```

Comment:
This file should be temporary, but allows for backwards compatibility with the current protocol and streaming set-up